ui/v2.5/src/components/Performers/PerformerDetails/PerformerOperationsPanel.tsx

```diff
@@ -13,11 +13,14 @@ export const PerformerOperationsPanel: React.FC<IPerformerOperationsProps> = ({
   performer,
 }) => {
   const Toast = useToast();
+  const intl = useIntl();
 
   async function onAutoTag() {
     try {
       await mutateMetadataAutoTag({ performers: [performer.id] });
-      Toast.success({ content: "Started auto tagging" });
+      Toast.success({
```

Comment:
Beat me to it

https://github.com/stashapp/stash/pull/1927/files#diff-74e92961bf92654d420b28e6c909729ee5c396b44af0eaa9b3b44c8137bbf526R24